Src/cql-lm/schema/elm/library.xsd

```diff
@@ -1,171 +1,176 @@
 <?xml version="1.0"?>
 <!-- This file defines the root library element and type for the ELM schema. -->
```

Comment:
I can't tell what changed or did not change in this file (due to all the reformatting).  Did anything change?  If so, what?